python/lsst/ts/externalscripts/maintel/take_twilight_flats_lsstcam.py

```diff
@@ -0,0 +1,134 @@
+
+from ..base_take_twilight_flats import BaseTakeTwilightFlats
+
+
+class TakeTwilightFlatsLSSTCam(BaseTakeTwilightFlats):
+    """Specialized script for taking Twilight flats with LSSTCam."""
+
+    def __init__(self, index):
+        super().__init__(index=index, descr="Take Twilight flats with LSSTCam.")
+
+        self.mtcs = None
+        self.LSSTcam = None
+
+        self.instrument_name = "LSSTCam"
+
+    @property
+    def tcs(self):
+        return self.mtcs
+
+    async def configure_tcs(self) -> None:
+        """Handle creating the ATCS object and waiting remote to start."""
+        if self.mtcs is None:
+            self.log.debug("Creating MTCS.")
+            self.mtcs = MTCS(
+                domain=self.domain,
+                log=self.log,
+            )
+            await self.mtcs.start_task
+        else:
+            self.log.debug("MTCS already defined, skipping.")
+
+    @property
+    def camera(self):
+        return self.lsstcam
+
+    async def configure_camera(self) -> None:
+        """Handle creating the camera object and waiting remote to start."""
+        if self.lsstcam is None:
+            self.log.debug("Creating Camera.")
+            self.lsstcam = LSSTCam(
+                self.domain, intended_usage=LSSTCamUsages.TakeImage, log=self.log
+            )
+            await self.lsstcam.start_task
+        else:
+            self.log.debug("Camera already defined, skipping.")
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_externalscripts/take_twilight_flats_lsstcam.yaml
+            title: TakeTwilightFlatsLSSTCam v1
+            description: Configuration for TakeTwilightFlatsLSSTCam.
+            type: object
+            properties:
+              filter:
+                description: Filter name or ID.
+                anyOf:
+                  - type: string
+                  - type: integer
+                    minimum: 1
+                  - type: "null"
+                default: "r_03"
+            additionalProperties: false
+        """
+        schema_dict = yaml.safe_load(schema_yaml)
+
+        base_schema_dict = super(TakeTwilightFlatsLSSTCam, cls).get_schema()
+
+        for prop in base_schema_dict["properties"]:
+            schema_dict["properties"][prop] = base_schema_dict["properties"][prop]
+
+        return schema_dict
+
+    async def get_sky_counts(self) -> float:
+        """Abstract method to get the median sky counts from the last image.
+
+        Returns
+        -------
+        float
+            Sky counts in electrons.
+        """
+        # query consDB with util once such a function exists
+        sky_counts = 0
+        return sky_counts
+
+    def get_instrument_name(self) -> str:
+        """Get instrument name.
+
+        Returns
+        -------
+        instrument_name: `string`
+        """
+        return self.instrument_name
```

Comment:
Same as above about removing `instrument_name` and just return the value directly here.